.eslintrc.js

```diff
@@ -1,17 +1,61 @@
 const { overrides } = require('@netlify/eslint-config-node')
 
 module.exports = {
-  extends: '@netlify/eslint-config-node',
+  extends: ['@netlify/eslint-config-node', 'plugin:node/recommended'],
```

Comment:
Can we share these configs via `@netlify/eslint-config-node`? Is this planned for another PR?

Reply:
Yes, the goal of this PR (and upcoming PRs) is to put those in `@netlify/eslint-config-node`.

We could either:
  1. Add those to `@netlify/eslint-config-node`, make a release, then upgrade in CLI/Netlify Build
  2. Or add the rules to CLI/Netlify Build, then do `1.`

I went with `2.` because it allows us to discuss those rules, and see their impact to the repositories before we commit to them in `@netlify/eslint-config-node`. What do you think?

Reply:
Sounds good to me